### PR TITLE
Indicator Layouts cherry pick

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
Ready

## Description
Cherry picked 33 files from master (the files that were created in https://github.com/demisto/content/pull/4084)

### CODE REVIEW

File: Scripts/IndicatorRelatedIncientBySeverity/IndicatorRelatedIncientBySeverity.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  name: indicator
+  required: true
+  secret: false
+commonfields:
+  id: IndicatorRelatedIncientBySeverity
+  version: -1
+enabled: true
+name: IndicatorRelatedIncientBySeverity
+script: '-'
+system: false
```
Comment (yuvalbenshalom) [inline Scripts/IndicatorRelatedIncientBySeverity/IndicatorRelatedIncientBySeverity.yml]: remove system

File: Scripts/NumberOfPhishingAttemptPerUser/NumberOfPhishingAttemptPerUser.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+comment: Shows a bar chart of the number of incident the 'To' and 'From' email addresses
+  appeared on
+fromversion: 5.0.0
+commonfields:
+  id: NumberOfPhishingAttemptPerUser
+  version: -1
+enabled: true
+name: NumberOfPhishingAttemptPerUser
+script: '-'
+system: false
```
Comment (yuvalbenshalom) [inline Scripts/NumberOfPhishingAttemptPerUser/NumberOfPhishingAttemptPerUser.yml]: remove system

File: Scripts/PositiveDetectionsVSDetectionEngines/PositiveDetectionsVSDetectionEngines.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  name: indicator
+  required: true
+  secret: false
+commonfields:
+  id: PositiveDetectionsVSDetectionEngines
+  version: -1
+enabled: true
+name: PositiveDetectionsVSDetectionEngines
+script: '-'
+system: false
```
Comment (yuvalbenshalom) [inline Scripts/PositiveDetectionsVSDetectionEngines/PositiveDetectionsVSDetectionEngines.yml]: remove system

**Comment by yardensade** [comment]
Alex's team will need to manually add reviewed RN to the release draft, is he aware of that?

**Comment by yardensade** [comment]
@kirbles19

**Comment by kirbles19** [comment]
@DeanArbel @yardensade All changelog items added to 19.8.1 RNs manually.

**Comment by yardensade** [comment]
This happens in the build atm, do we know why?
image

**Comment by yardensade** [comment]
@DeanArbel @yuvalbenshalom

**Pull request merged**